x/evm/types/contracts.go

```diff
@@ -1,8 +1,22 @@
-	burnable         = "6080604052348015600f57600080fd5b506040516100c23803806100c2833981016040819052602c916089565b6040516308a1eee160e01b8152600481018290526001600160a01b038316906308a1eee190602401600060405180830381600087803b158015606d57600080fd5b505af11580156080573d6000803e3d6000fd5b50600092505050ff5b60008060408385031215609b57600080fd5b82516001600160a01b038116811460b157600080fd5b602093909301519294929350505056fe"
+import (
+	"io/ioutil"
+	"encoding/json"
+)
+
+func getBytecodeFromArtifact(filename string) string {
+    var compiled map[string]interface{}
+    byteValue, _ := ioutil.ReadFile("artifacts/" + filename)
+
+    json.Unmarshal(byteValue, &compiled)
+
+    return compiled["bytecode"].(string)
+}
+
+var (
+	singlesigGateway = getBytecodeFromArtifact("AxelarGatewaySinglesig.json")
+	multisigGateway  = getBytecodeFromArtifact("AxelarGatewaySinglesig.json")
+	token            = getBytecodeFromArtifact("ERC20.json")
+	burnable         = getBytecodeFromArtifact("Burner.json")
```

Comment:
```suggestion
	singlesigGateway = getBytecodeFromArtifact("AxelarGatewayProxySinglesig.json")
	multisigGateway  = getBytecodeFromArtifact("AxelarGatewayProxyMultisig.json")
	token            = getBytecodeFromArtifact("BurnableMintableCappedERC20.json")
	burnable         = getBytecodeFromArtifact("Burner.json")
```
@sammy1991106 can confirm

Reply:
When we refer to the gateway contracts on chain, we're actually talking about the proxy contract addresses.

Reply:
@milapsheth thank you for the feedback! I just wanted to mention that we are importing not the address but the bytecode here (smart contract as a binary code). I am going to sync up with Sammy to confirm those details.

Reply:
confirm that @milapsheth was right.

Reply:
Thank you guys for pointing this out!